src/main/java/duke/main/Parser.java

```diff
@@ -0,0 +1,66 @@
+package duke.main;
+
+import duke.command.Command;
+import duke.command.CreateCommand;
+import duke.command.DeleteCommand;
+import duke.command.ExitCommand;
+import duke.command.ReadCommand;
+import duke.command.UpdateCommand;
+import duke.exception.DukeException;
+
+/**
+ * Represents a parser for user input. A Parser is able to understand user input and validates
+ * against poor command formats from users to direct appropriate Command response.
+ */
+public class Parser {
+
+    /**
+     * Returns appropriate Command object as specified by the user.
+     * Flags error when user command is invalid or error occurs.
+     *
+     * @param command  The input as specified by the user.
+     * @throws DukeException  Error while executing command.
+     */
+    public static Command parse(String command) throws DukeException {
+        String[] commandArr = command.split(" ");
+        String directive = commandArr[0];
+        if (directive.equals("list")) {
```

Comment:
Changing a chain of `if-else` statements to `switch-case` would make the code cleaner.